app/about/page.tsx

```diff
@@ -28,80 +31,94 @@ const filePath = path.join("content/about", fileName)
+                Brown University and its affiliated hospitals, complemented by
+                training initiatives to develop the next cohort of junior
+                investigators.
+              </p>
+            </CardContent>
+          </Card>
+        </section>
 
-      {/* People */}
-      <div className="content-wrapper py-16 sm:py-24">
-        <SectionHeader title="People" align="center"></SectionHeader>
-        <div className="flex justify-center">
-          <div className="flex flex-wrap justify-center gap-y-6 xs:w-1/2">
-            {pageContent?.data?.map((person: peopleTypes) => (
-              <div key={person.name}>
-                <CardWithImage
-                  imagePath={imagePath(person?.image)}
-                  hoverImagePath={imagePath(
-                    person?.image.replace("main", "hover")
-                  )}
-                  name={person?.name}
-                  title={person?.title}
-                />
-              </div>
-            ))}
+        {/* People */}
+        <div className="content-wrapper py-16 sm:py-24">
+          <SectionHeader title="People" align="center"></SectionHeader>
+          <div className="flex justify-center">
+            <div className="flex flex-wrap justify-center gap-y-6 xs:w-1/2">
+              {pageContent?.data?.map((person: peopleTypes) => (
+                <div key={person.name}>
+                  <CardWithImage
+                    imagePath={imagePath(person?.image)}
+                    hoverImagePath={imagePath(
+                      person?.image.replace("main", "hover")
+                    )}
+                    name={person?.name}
+                    title={person?.title}
+                  />
+                </div>
+              ))}
+            </div>
           </div>
         </div>
-      </div>
 
-      {/* Careers */}
-      <section id="careers" className="content-wrapper py-24 bg-gray-100">
-        <SectionHeader title="Careers" align="center" />
-        <Card className="w-full border-none shadow-none rounded-none">
-          <CardContent className="max-w-[1440px] mx-auto max-h-[600px] flex justify-center items-center px-6 py-10">
-            <p className="text-neutral-700 text-2xl italic">
-              There are no positions open at the moment. Check back with us in
-              the future. We appreciate your interest!
-            </p>
-          </CardContent>
-        </Card>
-      </section>
-    </div>
-  )
+        {/* Careers */}
+        <section
+          id="careers"
+          className="content-wrapper py-24 px-36 bg-gray-100"
+        >
+          <SectionHeader title="Careers" align="center" />
+          <Card className="w-full border-none shadow-none rounded-none">
+            <CardContent className="max-w-[1440px] mx-auto max-h-[600px] flex flex-col content-center gap-8">
+              <Suspense fallback={<Spinner />}>
+                <Workday data={workdayData} />
```

Comment:
Since you updated the prop, i think you need to update it here too !

```suggestion
                <Workday careers={workdayData} />
```